src/components/authentication/LoginPage.jsx

```diff
@@ -71,18 +74,17 @@ class LoginPage extends Component {
     return <div>
       <div>
         <img className={classes.company} src={company} alt={'Galvanize Logo'}></img>
-        <button className={`${classes.button} ${classes.galvanize}`}>
-          <img src={galvanize} className={classes.logo} alt={'Galvanize Logo'}></img>
-          <div className={classes.divider}></div>
-          <p className={classes.text}>Sign in with Galvanize Account</p>
-        </button>
         <button className={`${classes.button} ${classes.microsoft}`}>
           <img src={microsoft} className={classes.logo} alt={'Microsoft Logo'}></img>
           <div className={classes.divider}></div>
           <p className={classes.text}>Sign in with Microsoft Account</p>
         </button>
-        {/* <p className={classes.help}>Trouble signing in?<span className={classes.highlight}>Recover account</span></p> */}
-        <div className={classes.footer}><p className={classes.license}>© 2019 ACL Services Ltd. dba Galvanize. All Rights Reserved.</p></div>
+        <Typography align="center">or</Typography>
+        <button className={`${classes.button} ${classes.galvanize}`}>
```

Comment:
I think I changed this to using `clsx` at some point, I can make that changes if you want me to!

Reply:
What is clsx for? Compiling multiple stylesheets?